src/mbgl/renderer/tile_pyramid.cpp

```diff
@@ -297,13 +300,6 @@ std::unordered_map<std::string, std::vector<Feature>> TilePyramid::queryRendered
 
     mapbox::geometry::box<double> box = mapbox::geometry::envelope(queryGeometry);
 
-    std::vector<std::reference_wrapper<const RenderTile>> sortedTiles{ renderTiles.begin(),
-                                                                       renderTiles.end() };
-    std::sort(sortedTiles.begin(), sortedTiles.end(), [](const RenderTile& a, const RenderTile& b) {
-        return std::tie(a.id.canonical.z, a.id.canonical.y, a.id.wrap, a.id.canonical.x) <
-            std::tie(b.id.canonical.z, b.id.canonical.y, b.id.wrap, b.id.canonical.x);
-    });
```

Comment:
This sort function sorts differently from the standard UnwrappedTileID's `operator<` function: It sorts by (z, y, wrap, x), while `UnwrappedTileID::operator<` sorts by (wrap, z, x, y). I'm not sure why we sort this way here, but might be worth investigating how a different sort order affects feature querying.

Reply:
Good point, thanks for pointing that out!